care/facility/api/viewsets/patient.py

```diff
@@ -1,8 +1,10 @@
 import datetime
 import json
 from json import JSONDecodeError
+from config.celery_app import app
 
 from django.contrib.postgres.search import TrigramSimilarity
+from django.core.validators import validate_email
```

Comment:
Codacy found an issue: [third party import "from django.core.validators import validate_email" should be placed before "from config.celery_app import app"](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5765271)